test/e2e/pages/index.js

```diff
@@ -15,6 +21,9 @@ const dashboardPage = new DashboardPage()
 
 export {
   page,
+  allocationCancelPage,
+  allocationCriteriaPage,
```

Comment:
These could be combined to follow the same pattern for moves: `createAllocationPage`?

Reply:
But why exactly? The split between files helps keep them small and tidy, and the common methods are in `_allocation`. What are in your opinion the drawbacks of such approach?

Reply:
It was just a suggestion to follow the pattern for creating a move. If we ended up with more steps we might up with lots of pages that was the only reason I was suggesting that.

My comments aren't always things we must do, they're talking points. You know more about this piece of code so it's just to create a discussion around changes and get a shared goal moving forward.

The pattern of `_allocation` isn't something I think we want to maintain though. I feel the things added to `_allocation` could easily fit within a page object model so rather than maintain two patterns (or continue to add to the two existing patterns) we should strive to use one approach as it will be easier to maintain long term. @solidgoldpig and I discussed moving some things out of `_move` to a more page object approach but as is often the case we had other delivery pressures.

So in summary, I think it would be better to keep these changes to using one approach rather than two.

Reply:
Ok, I acknowledge it was a talking point -- however you formally requested changes, so it wasn't immediately evident what was requested.
I am ok to reincorporate the three files (the two pages files, plus `_allocation`) as long as the pattern makes sense. However, whilst I appreciate that we want to move away from the status quo, I need to be able to appreciate its drawbacks. Currently, I see none, unless we want to genericise most actions across the board (e.g. `moves`,  `allocation` and whatever new comes up). Which might be a good idea, but way outside the scope of this PR, which meant to simply complement the allocations feature.

In short -- what do you suggest I do?

Reply:
Sorry for the confusion. The requested changes were to things that weren’t needed or I felt could be updated to keep inline with the rest of the codebase so that it remains maintainable and consistent. 

This particular line isn’t a big issue as it keeps to the concept of using page objects to structure tests and was added as a comment, not a change request.  

My concern is with the addition of the `_allocation` file as this doesn’t follow the status quo, this in fact starts to move away from it and will it harder for us to maintain the page object pattern moving forward. The desired goal is to use page objects which are a widely understood concept within end to end testing and come with benefits such as isolating where the tests interact with elements of the UI instead of having those interactions in multiple places. 

So in short, can we move the methods from `_allocation` to a page object.

Reply:
Ok, have a look now, if it seems better.